### DISCUSSION TITLE
Add issues from organisations to personal projects?

### DISCUSSION THREAD

**Discussion by Guite** [community/community#8369] [Projects and Issues]
Hello,

first, the new issues beta is amazing work so far!

Currently it seems not possible to add issues to a personal project if they belong to a organisation's repository. Is that correct or did I miss something obvious?
That would be a very handy feature as it would simplify planning cycles and work shares across multiple organisations.
For example, having a board/table that combines issues from repository A (from the user) and repository B (from a user's organisation).

Thought even further, this is not necessarily limited to organisations for which a user has a membership because there are also additional contributions and tasks a user may be working on.

**Comment by louwers** [comment]
I agree this would be useful.

I want to have an overview of all my tasks and right now this does not seem to be possible if they are part of multiple projects.
